go.mod

```diff
@@ -45,6 +45,7 @@ require (
 	k8s.io/apiserver v0.17.0
 	k8s.io/client-go v11.0.1-0.20190409021438-1a26190bd76a+incompatible
 	k8s.io/component-base v0.17.0
+	k8s.io/helm v2.16.1+incompatible
```

Comment:
I cant see dependencies on that ?

Reply:
I am also not sure where this comes from. I manually removed it from the file, rebuild and it didn't come back. :confused: